README.md

```diff
@@ -8,6 +8,15 @@ This codebase is a [Lerna monorepo](https://github.com/lerna/lerna).
 
  For a list of all modules in this repository, see [packages](https://github.com/transmute-industries/sidetree.js/tree/master/packages).
 
+### Sidetree Based DID Methods
+
+| Method  | Spec  | Ledger  |
+|---|---|---|
+| did:ion  | [spec](https://github.com/decentralized-identity/ion) | Bitcoin  |
```

Comment:
This does not point to the spec, nor does the trustblock link

Reply:
there is no method spec for ion... see https://w3c.github.io/did-spec-registries/

Reply:
I think I have fixed these.